components/Navbar.tsx

```diff
@@ -19,7 +20,7 @@ export default function Navbar() {
                             alt='Dynamic Image'
                         />
                         <Text fontSize='xl' fontWeight='bold' ml={2}>
-                            Elegangster Barber Studio
+                            {businessInfo.name}
```

Comment:
💭  you need to account for the isLoading property too, since now you're fetching this data from the back and the display of the name may not be immediately sometimes 👀